cdap-ui/app/features/admin/templates/overview.html

```diff
@@ -5,12 +5,9 @@
         Welcome CDAP Admin!
       </h3>
     </div>
-    <div class="panel-body" ng-show="isEnterprise">
-      <p>This is the control console for CDAP (Enterprise mode). Here, you can view and manage your system configuration and define and manage namespaces, entities and access permissions to keep your team's work organized and secure.</p>
-      <p>Begin by configuring your default namespace.</p>
-    </div>
-    <div class="panel-body" ng-hide="isEnterprise">
-      <p>This is the control console for CDAP (Standalone mode).</p>
+    <div class="panel-body">
+      <p>This is the control console for CDAP. Here, you can view and manage your system configuration and define and manage namespaces and entities.
```

Comment:
Here, I removed the part about access permissions, because that is not true. Ok with Nitin.